Sources/MapboxNavigation/NavigationView.swift

```diff
@@ -144,7 +144,8 @@ open class NavigationView: UIView {
         }
     }
     
-    lazy var wayNameView: WayNameView = {
+    // :nodoc:
+    public lazy var wayNameView: WayNameView = {
```

Comment:
Will allow to change `WayNameView` position and hide/show it whenever it's contained in `NavigationView`. Related to https://github.com/mapbox/mapbox-navigation-ios/pull/4004.